website/pages/docs/features/color-mode.mdx

```diff
@@ -119,6 +119,7 @@ file.
 ```jsx live=false ln={10}
```

Comment:
I assume you'd need to update the `ln={10}` since that is used to highlight a specific area in the code.

Reply:
yes, I will do the change. Thanks for pointing it out.